### PR TITLE
Set new password

### PR DESCRIPTION
**Pull request by MuhammadKhalilzadeh**

### CODE REVIEW

File: Client/src/Components/DashboardProgressBars/index.jsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { BarChart } from '@mui/x-charts/BarChart';
+// import { BarChart } from '@mui/x-charts/BarChart';
```
Comment (veyselboybay) [inline Client/src/Components/DashboardProgressBars/index.jsx:1]: Can you uncomment the import statement? It throws an error.

**Review by popnfresh234 — COMMENTED** [review]
Aside from the commented out code @veyselboybay mentioned looks good to me

File: Client/src/Components/DashboardProgressBars/index.jsx (hunk 1 above)
Comment (MuhammadKhalilzadeh) [reply Client/src/Components/DashboardProgressBars/index.jsx]: I just un-commented it.
The first time I pulled the changes it was causing some errors related to some dependencies, so I commented it to proceed, but it's uncommented now.

Comment (popnfresh234) [reply Client/src/Components/DashboardProgressBars/index.jsx]: Make sure to run `npm install` after pulling if new node modules have been added.

If there are further dependency errors you can try deleting `node_modules` and `pacakge-lock.json` and rerunning `npm install` to reinstall all node modules. 

Those two operations should resolve dependency issues! :rocket:

**Review by popnfresh234 — APPROVED** [review]
Good on my end :+1:

**Comment by veyselboybay** [comment]
Looks good @MuhammadKhalilzadeh , thanks!

**Pull request merged**